client.cc

```diff
@@ -35,178 +37,81 @@ using std::vector;
 using std::pair;
 using grpc::Channel;
 using grpc::ClientContext;
+using grpc::ClientReader;
 using grpc::Status;
-using supplyfinder::HelloRequest;
-using supplyfinder::HelloReply;
-using supplyfinder::Supplier;
-using supplyfinder::Vendor;
-using supplyfinder::FoodID;
-using supplyfinder::VendorInfo;
-using supplyfinder::InventoryInfo;
+using namespace supplyfinder;
```

Comment:
Generally, using an entire namespace is not a good idea.  You should either use specific items in a namespace at the top or fully qualify namespace paths when you use the item.